lib/rubocop/token.rb

```diff
@@ -14,7 +14,8 @@ def self.from_parser_token(parser_token)
     def initialize(pos, type, text)
       @pos = pos
       @type = type
-      @text = text
+      # Parser token "text" may be a Fixnum
+      @text = text.is_a?(String) ? text : text.to_s
```

Comment:
Just updated the PR. Thanks again for the rigorous code review. Next time I won't make assumptions.